src/main/java/com/minecolonies/coremod/proxy/ClientProxy.java

```diff
@@ -264,8 +267,10 @@ public static void registerModels(@NotNull final ModelRegistryEvent event)
         }
 
         //Additionally we register an exclusion handler here;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)